### PR TITLE
localization for auth

### PR DESCRIPTION
**Pull request by trunghq-2593**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/???)

### CODE REVIEW

**Comment by trunghq-2593** [comment]
ready

File: resources/views/cms/register.blade.php [hunk 1]
```diff
@@ -11,83 +11,86 @@
...
-                <small class="text-danger rule">{{ $message }}</small>
-                @enderror
-                <div class="input-group mb-3">
-                    <input type="email" name="email" class="form-control" value="{{ old('email',$request->email ?? null) }}" placeholder="@lang('register.email')">
-                    <div class="input-group-append">
-                        <div class="input-group-text">
-                            <span class="fas fa-envelope"></span>
+                    @error('email')
+                    <small class="text-danger rule">{{ $message }}</small>
+                    @enderror
```
Comment (ghost) [inline resources/views/cms/register.blade.php]:
```suggestion
                    @error('email')
                        <small class="text-danger rule">{{ $message }}</small>
                    @enderror
```

File: resources/views/cms/register.blade.php [hunk 2]
```diff
@@ -11,83 +11,86 @@
...
-                <small class="text-danger rule">{{ $message }}</small>
-                @enderror
-                <div class="input-group mb-3">
-                    <input type="password" name="password" class="form-control" value="{{ old('password',$request->password ?? null) }}" placeholder="@lang('register.password')">
-                    <div class="input-group-append">
-                        <div class="input-group-text">
-                            <span class="fas fa-lock"></span>
+                    @error('password')
+                    <small class="text-danger rule">{{ $message }}</small>
+                    @enderror
```
Comment (ghost) [inline resources/views/cms/register.blade.php]: indent 4 spaces

File: resources/views/cms/register.blade.php [hunk 3]
```diff
@@ -11,83 +11,86 @@
...
-                <small class="text-danger rule">{{ $message }}</small>
-                @enderror
-                <div class="input-group mb-3">
-                    <input type="password" name="password_confirmation" class="form-control" value="{{ old('password',$request->password ?? null) }}" placeholder="@lang('register.confirm')">
-                    <div class="input-group-append">
-                        <div class="input-group-text">
-                            <span class="fas fa-lock"></span>
+                    @error('password')
+                    <small class="text-danger rule">{{ $message }}</small>
+                    @enderror
```
Comment (ghost) [inline resources/views/cms/register.blade.php]: tương tự :3 fix hết đi nhé

**Comment by trunghq-2593** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**